web/templates/settings/notification_settings.hbs

```diff
@@ -107,6 +107,24 @@
             {{> settings_save_discard_widget section_name="desktop-message-settings" show_only_indicator=(not for_realm_settings) }}
         </div>
 
+        <div class="banner-wrapper">
+            <div class="banner banner-warning desktop-setting-notifications">
+                <span class="banner-label">
+                    {{#tr}}
+                        Zulip needs your permission to enable desktop notifications for important messages.
```

Comment:
I feel like in this context, "Zulip needs your permission to enable desktop notifications." feels more natural, without the "for important messages"? @alya thoughts?

Reply:
Yes, that sounds like a good tweak to me.